code/modules/events/wormholes.dm

```diff
@@ -17,7 +17,7 @@
 		if((T.z in config.station_levels))
 			pick_turfs += T
 
```

Comment:
This bit up here could be better optimized - using `getturfs` for the z level on the main station, perhaps